api_src/components/schemas/transcript.yaml

```diff
@@ -0,0 +1,14 @@
+  # language=Markdown
+description: >
+  A transcript object included in the `transcripts` array.
+type: object
+properties:
+  url:
+    type: string
+    required: true
```

Comment:
When I run the linter, `required` isn't allowed here. Could you remove it?

Additionally, could you add a description field to `url` and `type`? You could borrow the description from the namespace docs. https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md#transcript

Reply:
Done! I also updated the `description` for the `transcript` schema itself to match the podcast-namespace docs.

Reply:
Do you think I should also add the optional `rel` and `language` properties now? https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md#attributes

Reply:
I'm not sure if the are parsed and stored for the API to return. @daveajones do you archive the `rel` and `language` field and will they be returned via the API if in the XML?